scripts/apps/search/directives/ItemList.js

```diff
@@ -668,6 +668,147 @@ export function ItemList(
                 }
             });
 
+            var MarkedDesksList = React.createClass({
```

Comment:
@gbbr I agree with your point. But if it is possible I'd like look into the ItemList file in a separate PR after this. Please let me know if that is is ok with you.